### PR TITLE
Fix package uninstall

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

**Comment by hvelarde** [comment]
there is a test failing but I don't know why; I tested it and the configlet is being removed.

seem to me there is still some stuff left behind.

File: src/collective/revisionmanager/Extensions/Install.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    util = sm.getUtility(IHistoryStatsCache)
+    sm.unregisterUtility(provided=IHistoryStatsCache)
+    del util
+    sm.utilities.unsubscribe((), IHistoryStatsCache)
+    del sm.utilities.__dict__['_provided'][IHistoryStatsCache]
+    del sm.utilities._subscribers[0][IHistoryStatsCache]
+    # logger.info('IHistoryStatsCache persistent utility successfully removed')
+
+
+def uninstall(portal, reinstall=False):
```
Comment (hvelarde) [inline src/collective/revisionmanager/Extensions/Install.py]: I tested manually and it was not working without the addition of this module but I can double check.

**Comment by hvelarde** [comment]
@mauritsvanrees thanks, you were right!

@tschorr I think this is ready to be merged, I can fix later the test on the configlet removal.

**Comment by tschorr** [comment]
This seems to be a problem with some versions pinned for 4.3.x and 5.0. After merging this PR, I will upgrade buildout-5.0.x.cfg to 5.0.6, that should make the original test pass for Plone 5.

**Comment by tschorr** [comment]
... done with
a4e82bcb447bc63a36d82901d4da1e02b6a32bae
dfe250ea83cd9542eadf1f81b558e2dfbf10cac3

The 4.3.x case maybe still needs some investigation.
Thank you!

**Comment by hvelarde** [comment]
maybe tomorrow I'll fix your development configurations in another PR.

**Pull request merged**